assets/js/phoenix_live_view/dom.js

```diff
@@ -331,28 +333,40 @@ let DOM = {
 
     if(feedbacks.length > 0){
       let selector = feedbacks.map(f => `[${phxFeedbackFor}="${f}"]`).join(", ")
-      DOM.all(container, selector, el => el.classList.add(PHX_NO_FEEDBACK_CLASS))
+      // permanently add the no feedback class to the elements, so that they are
+      // not removed on subsequent patches
+      // we only remove them once the elements are touched or the form is submitted,
+      // see showError
+      DOM.all(container, selector, el => JS.addOrRemoveClasses(el, [PHX_NO_FEEDBACK_CLASS], []))
     }
   },
 
-  resetForm(form, phxFeedbackFor){
+  resetForm(form, phxFeedbackFor, phxFeedbackGroup){
     Array.from(form.elements).forEach(input => {
       let query = `[${phxFeedbackFor}="${input.id}"],
```

Comment:
Btw, wasn't the plan to stop querying it based on the ID? Perhaps now is a good time to remove it?

Reply:
The changelog says we removed this feature on v0.19.0 :)

Reply:
removed